lte/cloud/go/services/lte/servicers/builder_servicer.go

```diff
@@ -92,7 +98,7 @@ func (s *builderServicer) Build(ctx context.Context, request *builder_protos.Bui
 
 	federatedNetworkConfigs, err := getFederatedNetworkConfigs(network.Type, cellularNwConfig.FegNetworkID, request)
 	if err != nil {
-		glog.Errorf("Failed to retrieve LTE_federated network config while building lte mconfig")
+		glog.Errorf("Failed to retrieve LTE_federated network config while building lte mconfig for gateway %s", request.GatewayId)
```

Comment:
question: why do we use glog.Error/Info in some places and glog.V() in others? Or can there be multiple verbose levels within INFO level?

Reply:
Yes, there are different levels. When `-v=0`  then we only see glog.Info and glog.Error

To see V(2) you need to set `-v=2`

I normally set info logs on places that runs once (for example read config file, start a service). This way I avoid too much verbosity.

The messages set as v=2 or v=4 are those use for debugging. 

In case of builder i set it as v=4 since builder runs every x seconds I didn't want to spam the log in case we are on v=2